docs/tutorials/taskrunner.ipynb

```diff
@@ -52,13 +56,31 @@
         "This step builds an entire FL experiment plan, along with the initial set of parameters that will be used in the experiment.\n",
         "We supply `localhost` as the aggregator address, for simulation purposes. The command below supports models of arbitrary input shapes and initializations (e.g. when using pre-trained models).\n",
         "\n",
-        "For a complete list of supported options, refer to [`fx plan initialize`](https://openfl.readthedocs.io/en/latest/reference/fx.plan.html#fx-plan-initialize)."
+        "For a complete list of supported options, refer to [`fx plan initialize`](https://openfl.readthedocs.io/en/latest/reference/fx.plan.html#fx-plan-initialize).\n",
+        "\n",
+        "Note: By default, this tutorial is based on the recommended mTLS communication setup. For local testing purposes, however, TLS can be disabled by:\n",
+        "\n",
+        "1. Setting `use_tls: false` in the FL plan\n",
+        "2. Skipping all certificate-related commands (incl. CA setup) in the following steps of the tutorial\n",
+        "3. Manually populating the collaborator names in `cols.yaml`. For example:"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "        collaborators:\n",
+        "        - collaborator1\n",
+        "        - collaborator2"
       ]
```

Comment:
I suggest moving this entire block into a separate section called "(Optional) Disable TLS for local testing", to avoid getting it mixed with the plan initialization step (which is the same in both TLS and non-TLS cases AFAIK)